ts/modules/timeline.src.ts

```diff
@@ -679,11 +681,11 @@ seriesType<Highcharts.TimelineSeries>('timeline', 'line',
                     pointMarkerOptions.width,
                     seriesMarkerOptions.width,
                     series.closestPointRangePx
-                ),
+                ) as any,
                 height = pick(
                     pointMarkerOptions.height,
                     seriesMarkerOptions.height
-                ),
+                ) as any,
```

Comment:
Here the same. This should not be done. Add the generic types and cast the last argument to `any`.